test/e2e/apimachinery/watchlist.go

```diff
@@ -53,12 +54,14 @@ var _ = SIGDescribe("API Streaming (aka. WatchList)", framework.WithSerial(), fu
 		featuregatetesting.SetFeatureGateDuringTest(ginkgo.GinkgoTB(), utilfeature.DefaultFeatureGate, featuregate.Feature(clientfeatures.WatchListClient), true)
 		stopCh := make(chan struct{})
 		defer close(stopCh)
+
 		secretInformer := cache.NewSharedIndexInformer(
 			&cache.ListWatch{
 				ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
 					return nil, fmt.Errorf("unexpected list call")
 				},
 				WatchFunc: func(options metav1.ListOptions) (watch.Interface, error) {
+					options = withWellKnownListOptions(options)
```

Comment:
nit: I believe you can remove this line and have the fxn in the params, eg:
```golang
return f.ClientSet.CoreV1().Secrets(f.Namespace.Name).Watch(context.TODO(), withWellKnownListOptions(options))
```
+1 to what @sttts mentioned though, as it is likely more readable to make this explicit via inlining the options (vs the helper)

Reply:
Another 2 cent: `withWellKnownListOptions`  is well known now to us, not to any ready in a few month. If we want a helper, call it `withWatchListLabel` or so. Then it is clear without looking it up what this does.